app/components/login-component/index.scss

```diff
@@ -0,0 +1,118 @@
+.container {
+  height: 100vh;
+}
+
+.inner-container {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  padding-top: 3em;
+}
+
+.login-container {
+  padding: 0em;
+  background-color: var(--white);
+  border-radius: 5px;
+  min-width: 26em;
+  box-shadow: 2px 2px 17px -1px rgba(0, 0, 0, 0.5);
+  margin-bottom: 3em;
+  margin-top: 3em;
+}
+
+.login-title {
+  font-size: 1.72rem;
+  text-align: center;
+  margin-top: 2em;
+}
+
+.login-footer {
+  margin-top: 3em;
+  border-top: 1px solid #efefef;
+  background: #f6f7f8;
+  padding: 1.5em 1em 1.6em;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 0.9em;
+  border-bottom-right-radius: 5px;
+  border-bottom-left-radius: 5px;
+}
+
+.register-link {
+  margin-left: 0.5em;
+}
+
+.action-container {
+  margin: 0 2em 3em 2em;
+}
+
+.input-group {
+  display: flex;
+  align-items: stretch;
+  flex-grow: 1;
+  margin-top: 1em;
+}
+
+.input {
+  align-items: center;
+  background-color: var(--white);
+  border: 1px solid #e6e6e6;
+  border-radius: 3px;
+  color: #6b6b6b;
+  display: inline-flex;
+  font-size: 14px;
+  height: 32px;
+  justify-content: flex-start;
+  line-height: 24px;
+  padding-left: 8px;
+  padding-right: 8px;
+  position: relative;
+  vertical-align: top;
+  box-shadow: inset 0 1px 2px rgba(107, 107, 107, 0.1);
+  max-width: 100%;
+  width: 100%;
+  &:active,
+  &:hover {
+    border: 1px solid #b3b3b3;
+    box-shadow: 0 0 1px grey;
+  }
+}
+
+.input-connect-label {
+  border-top-left-radius: 0;
+  border-bottom-left-radius: 0;
+}
+
+.input-label {
+  border-width: 1px 0 1px 1px;
+  border-style: solid;
+  border-color: #e6e6e6;
+  box-shadow: inset 0 1px 2px rgba(107, 107, 107, 0.1);
+  padding: 0 0.6em;
+  display: flex;
+  align-items: center;
+  background: #f7f7f7;
```

Comment:
to be consistence, all the new components are started with using style var as much as possible which is available in the theme.scss. Also, it's suggested to use em/rem values instead of px.

Reply:
i don't think we can use px for `border-width`, `box-shadow` or `border-radius` since it should not scale with font-size.